types/src/validator_verifier.rs

```diff
@@ -556,6 +556,22 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_empty_multi_signatures() {
+        let validator_signer = ValidatorSigner::random(TEST_SEED);
+        let dummy_struct = TestAptosCrypto("Hello, World".to_string());
+        let validator =
+            ValidatorVerifier::new_single(validator_signer.author(), validator_signer.public_key());
+
+        assert_eq!(
+            validator.verify_multi_signatures(&dummy_struct, &MultiSignature::empty()),
+            Err(VerifyError::TooLittleVotingPower {
```

Comment:
isn't it EmptySignature error expected? can we fill in the bitmask?

Reply:
This behavior is expected, added another test for empty signature.